src/Subs/SubsDisplay.js

```diff
@@ -16,7 +16,7 @@ export default function SubsDisplay({
 
   const q = query.replace(/[\\^$.*+?()[\]{}|]/g, "\\$&");
   // TODO: Display chunks. May get query, which should be highlighted if exists
```

Comment:
What does this comment mean again? Is it still relevant? I can remove it as a part of this PR.

Reply:
This is a good point. My understanding is that this refers to the functionality of the search function, which does exist now. So this should indeed be removed.